.github/workflows/build.yml

```diff
@@ -23,9 +23,9 @@ jobs:
         include:
           - os: buildjet-4vcpu-ubuntu-2204-arm
             arch: arm64
-          - os: buildjet-4vcpu-ubuntu-2004
+          - os: ubuntu-20.04
```

Comment:
Can we use both?

Reply:
sure

Reply:
But here you switch from buildjet to a regular one?

Reply:
So in this particular case it won't be that easy, because we're running matrix and we have runs-on parameter, and we have both amd64 and arm arch to avoid cross-compiling. We can add a second option as fallback in runs-on parameter, but it'll mess up arm runner. I think it makes sense to switch to GH-hosted runners here, since it'll free up buildjet runners a bit

Reply:
ok, let's try